app/templates/main/index.html

```diff
@@ -8,25 +8,166 @@
     <meta charset="utf-8">
     <meta name="csrf-token" content="{{ csrf_token() }}">
 
-    <div class="ui container">
-        <h1>Maps4All</h1>
-        <a href="{{ url_for('suggestion.suggest') }}">Suggest New Resource</a>
-        <input id="pac-input" class="controls" type="text"
-               placeholder="Enter a location">
-        <div id="view-selector" class="controls">
-          <input type="radio" name="toggle" id="map_view" checked="checked">
-          <label for="map_view">Map View</label>
+    <div class="main col-md-7 col-sm-12">
+      <div class="header">
+        <div id="logo">
+          <a href="http://www.asylumconnect.org/"><img style="width: 100px;" src="/static/asylum-assets/images/logo.jpeg" alt="AsylumConnect"></a>
+        </div>
+        <div class="left-header">
+          <div id="google_translate_element" class="translate"></div>
+          <div id="links" class="hidden-xs">
+            <a href="http://www.asylumconnect.org/our-cause/">ABOUT</a>
+            <a href="http://www.asylumconnect.org/follow-us-subscribe/">JOIN</a>
+            <a href="http://www.asylumconnect.org/resources/">RESOURCES</a>
+            <a href="http://www.asylumconnect.org/contact-us/">CONNECT</a>
+          </div>
+        </div>
+      </div>
+      <div class="filters">
+        <!-- data-spy="affix" data-offset-top="280" -->
+        <h3 id="location-query">Find LGBTQ- and asylum-friendly resources in
+          <div class="btn-group">
```

Comment:
I think the feature is small enough to just keep it here for sake of keeping things simple.